istack-commons/runtime/pom.xml

```diff
@@ -90,7 +90,7 @@
                             *
                         </Export-Package>
                         <Import-Package>
-                            javax.activation;version=!,
+                            jakarta.activation;version=!,
                             *
```

Comment:
Reason why this was there was to allow javax.activation package coming from the runtime (typically but not always under version 0.0.0.0), since "jakarta.activation" package is not part of any existing runtime (yet?),  this explicit configuration is not needed.